setup.py

```diff
@@ -58,7 +58,7 @@ def read(fname):
     "pytest-runner>=4.4",
     "pytest-qt>=3.3.0",
     "quilt3>=3.1.12",
-    "Sphinx>=2.0.0b1,<3",
+    "Sphinx>=2.0.1b1,<3",
```

Comment:
this should not have changed, please revert

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/106/commits/8ca00d3ee8a5cbc66bb365bafa29efd0be7ef0e6